.gitignore

```
@@ -0,0 +1,2 @@
+.vscode
+node_modules
```

```suggestion
node_modules

```
@AbdoulNuru Add a new line

---

@hakoemmy Thanks